services/fileServices/MdPageServices/__tests__/subcollectionPageService.spec.js

```diff
@@ -0,0 +1,238 @@
+        fileName,
+        directoryName,
+      })
+    })
+    it("Creating a page which specifies a subcollection in the front matter removes the third_nav_title parameter", async () => {
+      const mockFrontMatterWithSubcollection = {
+        ...mockFrontMatter,
+        third_nav_title: "mock-third-nav",
+      }
+      await expect(
+        service.create(reqDetails, {
+          fileName,
+          collectionName,
+          subcollectionName,
+          content: mockContent,
+          frontMatter: mockFrontMatterWithSubcollection,
+        })
+      ).resolves.toMatchObject({
+        fileName,
+        content: { frontMatter: mockFrontMatter, pageBody: mockContent },
+        sha,
+      })
+      expect(convertDataToMarkdown).toHaveBeenCalledWith(
+        mockFrontMatter,
+        mockContent
+      )
+      expect(mockCollectionYmlService.addItemToOrder).toHaveBeenCalledWith(
+        reqDetails,
+        collectionYmlObj
+      )
+      expect(mockGithubService.create).toHaveBeenCalledWith(reqDetails, {
+        content: mockMarkdownContent,
+        fileName,
+        directoryName,
+      })
+    })
+  })
+
+  describe("Read", () => {
+    mockGithubService.read.mockReturnValue({
+      content: mockMarkdownContent,
+      sha,
+    }),
+      it("Reading pages works correctly", async () => {
+        await expect(
+          service.read(reqDetails, {
+            fileName,
+            collectionName,
+            subcollectionName,
+          })
+        ).resolves.toMatchObject({
+          fileName,
+          content: { frontMatter: mockFrontMatter, pageBody: mockContent },
+          sha,
+        })
+        expect(retrieveDataFromMarkdown).toHaveBeenCalledWith(
+          mockMarkdownContent
+        )
+        expect(mockGithubService.read).toHaveBeenCalledWith(reqDetails, {
+          fileName,
+          directoryName,
+        })
+      })
+  })
+
+  describe("Update", () => {
+    const oldSha = "54321"
+    mockGithubService.update.mockReturnValue({ newSha: sha })
+    it("Updating page content works correctly", async () => {
+      await expect(
+        service.update(reqDetails, {
+          fileName,
+          collectionName,
+          subcollectionName,
+          content: mockContent,
+          frontMatter: mockFrontMatter,
+          sha: oldSha,
+        })
+      ).resolves.toMatchObject({
+        fileName,
+        content: { frontMatter: mockFrontMatter, pageBody: mockContent },
+        oldSha,
+        newSha: sha,
+      })
+      expect(convertDataToMarkdown).toHaveBeenCalledWith(
+        mockFrontMatter,
+        mockContent
+      )
+      expect(mockGithubService.update).toHaveBeenCalledWith(reqDetails, {
+        fileName,
+        directoryName,
+        fileContent: mockMarkdownContent,
+        sha: oldSha,
+      })
+    })
+  })
+
+  describe("Delete", () => {
+    it("Deleting pages works correctly", async () => {
+      await expect(
+        service.delete(reqDetails, {
```

Comment:
Should also expect `mockCollectionYmlService.deleteItemFromOrder` to be called

Reply:
Resolved in https://github.com/isomerpages/isomercms-backend/pull/259/commits/709a7fc18fdc815e314af00330b0e72062b23059